gen/elerning_mu_ff

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use perunServicesInit;
+use perunServicesUtils;
+use File::Basename;
+
+our $SERVICE_NAME = basename($0);
+our $PROTOCOL_VERSION = "3.0.1";
+my $SCRIPT_VERSION = "3.0.2";
+
+perunServicesInit::init;
+my $DIRECTORY = perunServicesInit::getDirectory;
+my $data = perunServicesInit::getHashedHierarchicalData;
+
+#Constants
+our $A_USER_LOGIN;          *A_USER_FACILITY_LOGIN =            \'urn:perun:user:attribute-def:def:login-namespace:mu';
```

Comment:
Names must match between `$A_USER_LOGIN` and `*A_USER_FACILITY_LOGIN`. Also please align it in columns with other definitions below.